src/tile/tile.scss

```diff
@@ -197,7 +197,6 @@
   }
 
   ~ .iui-thumbnail-icon {
-    z-index: 1;
```

Comment:
I think this was added for an edge case. @FlyersPh9 Do you remember?

Reply:
I thought that this had to do with a tile that has a placeholder icon & a thumbnail present.  I quickly scanned `tile.html` and didn't see any demos that are testing that scenario.  I'm thinking we may be able to remove the `z-index` as long as the HTML is in this order (`.iui-thumbnail-icon` placed before `.iui-tile-thumbnail-picture`):
```html
<div class="iui-tile-thumbnail">
  <svg-imodel-hollow
    class="iui-thumbnail-icon"
    aria-hidden="true"
  ></svg-imodel-hollow>
  <div class="iui-tile-thumbnail-picture demo-photo"></div>
  <button
    aria-label="Info"
    type="button"
    class="iui-button iui-borderless iui-small iui-tile-thumbnail-quick-action"
  >
    <svg-info
      class="iui-button-icon"
      aria-hidden="true"
    ></svg-info>
  </button>
  <div class="iui-tile-thumbnail-badge-container">
    <div
      class="iui-badge"
      style="background-color: hsl(2, 90%, 83%);"
    >
      Draft
    </div>
  </div>
</div>
```